<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mission Editor</string>
  </property>
  <widget class="QWidget" name="dockContainer">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewBase"/>
    <addaction name="actionNewShip"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionTabOut"/>
    <addaction name="actionZoomToAll"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_Left"/>
    <addaction name="actionRotate_Right"/>
    <addaction name="separator"/>
    <addaction name="actionFlip_Horizontal"/>
    <addaction name="actionFlip_Vertical"/>
    <addaction name="separator"/>
    <addaction name="actionShrink"/>
    <addaction name="actionExtrude"/>
    <addaction name="actionUnion"/>
    <addaction name="actionFilter"/>
    <addaction name="actionAABB"/>
    <addaction name="actionConvexHull"/>
    <addaction name="actionReparent"/>
    <addaction name="separator"/>
    <addaction name="actionSelect"/>
    <addaction name="actionLasso"/>
    <addaction name="actionDraw"/>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView_Text"/>
    <addaction name="actionView_Points"/>
    <addaction name="actionView_Connections"/>
    <addaction name="actionView_Site"/>
    <addaction name="actionView_SiteContour"/>
    <addaction name="actionView_Walls"/>
    <addaction name="actionView_Compilation"/>
    <addaction name="actionView_Floor"/>
    <addaction name="actionView_Sections"/>
    <addaction name="actionView_Visibility"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolbarMain">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewBase"/>
   <addaction name="actionNewShip"/>
   <addaction name="separator"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_All"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolbarEdit">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTabOut"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionRotate_Left"/>
   <addaction name="actionRotate_Right"/>
   <addaction name="separator"/>
   <addaction name="actionFlip_Horizontal"/>
   <addaction name="actionFlip_Vertical"/>
   <addaction name="separator"/>
   <addaction name="actionShrink"/>
   <addaction name="actionExtrude"/>
   <addaction name="actionUnion"/>
   <addaction name="actionFilter"/>
   <addaction name="actionAABB"/>
   <addaction name="actionConvexHull"/>
   <addaction name="actionReparent"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionLasso"/>
   <addaction name="actionDraw"/>
   <addaction name="actionEdit"/>
  </widget>
  <widget class="QToolBar" name="toolbarView">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <addaction name="actionView_Text"/>
   <addaction name="actionView_Points"/>
   <addaction name="actionView_Connections"/>
   <addaction name="actionView_Site"/>
   <addaction name="actionView_SiteContour"/>
   <addaction name="actionView_Walls"/>
   <addaction name="actionView_Compilation"/>
   <addaction name="actionView_Floor"/>
   <addaction name="actionView_Sections"/>
   <addaction name="actionView_Visibility"/>
   <addaction name="separator"/>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockProperties">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockToolbox">
   <property name="windowTitle">
    <string>Toolbox</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="editor::MissionToolbox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>52</width>
          <height>1488</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Default</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewBase">
   <property name="icon">
    <iconset>
     <normaloff>:/art/blueprint.png</normaloff>:/art/blueprint.png</iconset>
   </property>
   <property name="text">
    <string>New Base</string>
   </property>
  </action>
  <action name="actionNewShip">
   <property name="icon">
    <iconset>
     <normaloff>:/art/blueprint.png</normaloff>:/art/blueprint.png</iconset>
   </property>
   <property name="text">
    <string>New Ship</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/art/disk.png</normaloff>:/art/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset>
     <normaloff>:/art/disk_saveas.png</normaloff>:/art/disk_saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset>
     <normaloff>:/art/folderOpen.png</normaloff>:/art/folderOpen.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoomToAll">
   <property name="text">
    <string>Zoom To All</string>
   </property>
   <property name="shortcut">
    <string>z</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/magnifier.png</normaloff>:/art/magnifier.png</iconset>
   </property>
  </action>
  <action name="actionTabOut">
   <property name="text">
    <string>Tab Out</string>
   </property>
   <property name="shortcut">
    <string>Tab</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/arrow_plus.png</normaloff>:/art/arrow_plus.png</iconset>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/cut.png</normaloff>:/art/cut.png</iconset>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/clipboard_copy.png</normaloff>:/art/clipboard_copy.png</iconset>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/clipboard_paste.png</normaloff>:/art/clipboard_paste.png</iconset>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  
  <action name="actionRotate_Left">
   <property name="text">
    <string>Rotate Left</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/arrow_clockwise.png</normaloff>:/art/arrow_clockwise.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="text">
    <string>Rotate Right</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/arrow_counter.png</normaloff>:/art/arrow_counter.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFlip_Horizontal">
   <property name="text">
    <string>Flip Horizontally</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/arrow_left.png</normaloff>:/art/arrow_left.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFlip_Vertical">
   <property name="text">
    <string>Flip Vertically</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/arrow_down.png</normaloff>:/art/arrow_down.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionShrink">
   <property name="text">
    <string>Shrink</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/shrink.png</normaloff>:/art/shrink.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExtrude">
   <property name="text">
    <string>Extrude</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/extrude.png</normaloff>:/art/extrude.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUnion">
   <property name="text">
    <string>Union</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/art/union.png</normaloff>:/art/union.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFilter">
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAABB">
   <property name="text">
    <string>AABB</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionConvexHull">
   <property name="text">
    <string>Convex Hull</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionReparent">
   <property name="text">
    <string>Reparent</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  
  <action name="actionView_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Connections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connections</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Site">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sites</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_SiteContour">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Contours</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Walls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Walls</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Compilation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Compilation</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Floor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Floor</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Sections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sections</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Visibility">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Visibility</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  
  <actiongroup name="Tools">
   <action name="actionSelect">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
   </action>
   <action name="actionLasso">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Lasso</string>
    </property>
    <property name="shortcut">
     <string>W</string>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
   </action>
   <action name="actionDraw">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
    <property name="shortcut">
     <string>D</string>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
   </action>
   <action name="actionEdit">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="shortcut">
     <string>E</string>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
   </action>
  </actiongroup>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>editor::MissionToolbox</class>
   <extends>QToolBox</extends>
   <header>toolboxPanel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
